patches/react-native-keyboard-controller+1.12.2+002+rn-75-fixes.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/react-native-keyboard-controller/android/src/main/java/com/reactnativekeyboardcontroller/extensions/ThemedReactContext.kt b/node_modules/react-native-keyboard-controller/android/src/main/java/com/reactnativekeyboardcontroller/extensions/ThemedReactContext.kt
+index 50252f0..28a70d6 100644
+--- a/node_modules/react-native-keyboard-controller/android/src/main/java/com/reactnativekeyboardcontroller/extensions/ThemedReactContext.kt
++++ b/node_modules/react-native-keyboard-controller/android/src/main/java/com/reactnativekeyboardcontroller/extensions/ThemedReactContext.kt
+@@ -13,7 +13,7 @@ val ThemedReactContext.rootView: View?
+ 
+ fun ThemedReactContext?.dispatchEvent(viewId: Int, event: Event<*>) {
+   val eventDispatcher: EventDispatcher? =
+-    UIManagerHelper.getEventDispatcherForReactTag(this, viewId)
++    UIManagerHelper.getEventDispatcherForReactTag(this!!, viewId)
```

Comment:
I think you can simply bump to `1.12.7` - that problem was fixed in `1.12.5`

The reason why we encounter this merge conflict again is because in main we bumped 1.12.2 -> 1.13.0 -> 1.12.2, you also bumped to `1.12.5`, then accepted remote resolution, and after downgrading to 1.12.2 you got merge conflicts

It's safe to bump to `1.12.5` or `1.12.7`. The `1.13.0` version had some issues affecting some functionality so was reverted to initial version 😔

Reply:
If we decide to merge it soon, I think we can bump those patches in some follow-up PR too.

Reply:
Ah, thanks for clarification @WoLewicki ! It absolutely makes sense what you are saying! Fingers crossed there will be no new issues and this PR finally can be merged 🤞 :shipit: 

Great job so fat 💪